### PR TITLE
Optimize dashboard queries

### PR DESCRIPTION
**Pull request by tadaskay**
1. Leaderboard
2. Node list
3. Some style improvements

mysteriumnetwork/node#1236

### CODE REVIEW

File: dashboard/static/build/css/custom.css [hunk 1]
```diff
@@ -491,7 +491,7 @@ body .container.body .right_col {
 body {
     color: #73879C;
     background: #2A3F54;
-    font-family: "Helvetica Neue", Roboto, Arial, "Droid Sans", sans-serif;
+    font-family: 'Ubuntu', sans-serif;
```
Comment (zolia) [inline dashboard/static/build/css/custom.css]: Ubuntu? does that even exists anywhere?

File: migrations/versions/e73480955f62_.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    conn = op.get_bind()
+    conn.execute("""
+
+    CREATE EVENT IF NOT EXISTS evt_refresh_dwh_leaderboard
+    ON SCHEDULE
+        EVERY 3 MINUTE
+    COMMENT 'Refresh monthly leaderboard'
+    DO
+    BEGIN
+        SET @service_type = 'openvpn',
```
Comment (zolia) [inline migrations/versions/e73480955f62_.py]: Why we account only openvpn service? Wireguard can run behind residential IP too.

File: dashboard/static/build/css/custom.css (hunk 1 above)
Comment (tadaskay) [reply dashboard/static/build/css/custom.css]: yes

File: migrations/versions/e73480955f62_.py (hunk 2 above)
Comment (tadaskay) [reply migrations/versions/e73480955f62_.py]: no idea, keeping existing functionality

**Pull request merged**